packages/astro/package.json

```
@@ -0,0 +1,34 @@
+{
+  "name": "@spotlightjs/astro",
+  "description": "Add spotlight to your Astro toolbar, for better debugging.",
+  "version": "0.0.0",
+  "type": "module",
+  "keywors": [
+    "astro",
+    "withastro",
+    "astro-component",
+    "analytics"
+  ],
+  "scripts": {
+    "dev": "astro dev",
+    "start": "astro dev",
+    "build": "astro build",
+    "preview": "astro preview",
+    "astro": "astro"
+  },
+  "main": "./src/index.tx",
+  "exports": {
+    ".": "./src/index.ts"
+  },
+  "dependencies": {
+    "@spotlightjs/core": "0.0.3",
+    "@spotlightjs/sidecar": "0.0.2",
```

Shouldn't these be 

```suggestion
    "@spotlightjs/core": "workspace:*",
    "@spotlightjs/sidecar": "workspace:*",
```

?

---

hmm not sure, as we publish this it should probably have the proper versions? I think pnpm should handle this properly, and changeset should also update these versions whenever we bump something (hopefully - let's see how that works out!)

---

> I think pnpm should handle this properly

Ahh, You mean in the sense that for local development it will take the workspace version instead of downloading the published versions? I [thought it was the other way around](https://pnpm.io/workspaces#publishing-workspace-packages), that when packing the tarball,`workspace:*` will be replaced with the package's version. But from reading [these docs](https://pnpm.io/workspaces#workspace-protocol-workspace) I now get the impression that it works both ways 😅 

So it's probably fine as is - let's give it a try!